iac/settings/prod.py

```diff
@@ -0,0 +1,24 @@
+"""Settings for production environment"""
+import os
+
+from aws_cdk import Environment
+
+from iac.settings.settings import DatabaseSettings, ProjectSettings, DjangoSettings
+
+PROD_ENV_ID = "prod"
+# PROD_SETTINGS = ProjectSettings(
```

Comment:
If we implement prod right now then we'd be spending extra $ for nothing...

Reply:
As discussed, makes good sense; let's just stub out what will need to happen, and comment and/or create future tasks. It occurs to me it would be a good exercise for me (with your guidance) to implement prod devops following your dev patterns, once dev is fully migrated....